test/test_nn.py

```diff
@@ -4571,21 +4571,22 @@ def test_pack_sequence_batch_sizes_throw(self):
             b = torch.tensor([1, 1, 1, 1, 1], device='cuda')
             input = nn.utils.rnn.PackedSequence(a, b)
 
-    def test_Transformer_cell(self):
+    def test_transformer_cell(self):
```

Comment:
you need to add a test to cover the batch_first argument with `True`

Reply:
Done in commit 248d44b

Reply:
Thanks, writing the test case helped uncover some potential problems. Now this has all been resolved as of commit 8bea5f0.